<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralLayerInspector</class>
 <widget class="QWidget" name="GeneralLayerInspector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
  background-origin: content;
  margin-top: 15px;
  font-weight: bold;
  font-size: 12px;
  font-family: helvetica;
  color: rgb(30,30,160);
  background-color: rgb(237,237,237);
  padding: 5px;
  border-radius: 4px;
  border: 1px solid rgb(130,130,130);
}
QGroupBox:!enabled {
  color: rgb(108,108,108);
}
QGroupBox::title {
  subcontrol-origin: 	margin;
  subcontrol-position: top left;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>18</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>18</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="grpInfo">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>General Information:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Filename:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="outFilename">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Nickname:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="inNickname">
        <property name="toolTip">
         <string>This is the name that is used when referring to the image in ITK-SNAP.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpOverlay">
     <property name="title">
      <string>Display Preferences:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1,0">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="0" column="1" colspan="2">
       <widget class="QWidget" name="grpOverlayChecks" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="btnUnpin">
           <property name="toolTip">
            <string>Render the image side by side with other image layers. This setting is best when working with multiple anatomical images at once.</string>
           </property>
           <property name="text">
            <string>Base layer</string>
           </property>
           <property name="icon">
            <iconset resource="../Resources/SNAPResources.qrc">
             <normaloff>:/root/icons8_unpin_12.png</normaloff>:/root/icons8_unpin_12.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="btnPin">
           <property name="toolTip">
            <string>Render the image as a semi-transparent overlay on top of other images. This setting is best for statistical maps.</string>
           </property>
           <property name="text">
            <string>Overlay</string>
           </property>
           <property name="icon">
            <iconset resource="../Resources/SNAPResources.qrc">
             <normaloff>:/root/icons8_pin_12.png</normaloff>:/root/icons8_pin_12.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QWidget" name="grpOpacity" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSpinBox" name="inOpacityValue">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="suffix">
            <string>%</string>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="inOpacity">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
           <property name="tickInterval">
            <number>25</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkVisible">
           <property name="text">
            <string>Visible</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Render image as:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblOpacity">
        <property name="text">
         <string>Overlay opacity:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Display order:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="btnUp">
           <property name="toolTip">
            <string>Place this image layer earlier in the workspace. This affects the order in which the layers are layed out and rendered.</string>
           </property>
           <property name="text">
            <string>Move up</string>
           </property>
           <property name="icon">
            <iconset resource="../Resources/SNAPResources.qrc">
             <normaloff>:/root/icons8_up_12.png</normaloff>:/root/icons8_up_12.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QToolButton" name="btnDown">
           <property name="toolTip">
            <string>Place this image layer later in the workspace. This affects the order in which the layers are layed out and rendered.</string>
           </property>
           <property name="text">
            <string>Move down</string>
           </property>
           <property name="icon">
            <iconset resource="../Resources/SNAPResources.qrc">
             <normaloff>:/root/icons8_down_12.png</normaloff>:/root/icons8_down_12.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpMulticomponent">
     <property name="title">
      <string>Multi-component Display:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnminimumwidth="110,0,0,0">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Display Mode:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QComboBox" name="inMode">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Multi-component images have more than one value for each voxel. These values may represent measurements taken at different time points, different chromatic components, etc. ITK-SNAP offers multiple ways to visualize this multi-dimensional information. You can choose to visualize just one component, or a summary function of the components, such as magnitude, average, or maximum. In the special case of three-component images, the components can be rendered as red, green and blue channels (RGB).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" alignment="Qt::AlignRight">
       <widget class="QCheckBox" name="chkAnimate">
        <property name="text">
         <string>Animate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblComponent">
        <property name="text">
         <string>Component:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="inComponent">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSlider" name="inComponentSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources/SNAPResources.qrc"/>
 </resources>
 <connections/>
</ui>
